src/skmultiflow/data/mixed_generator.py

```diff
@@ -6,16 +6,16 @@
 class MIXEDGenerator(Stream):
     r""" Mixed data stream generator.
 
-    This generator is an implementation of a data stream with abrupt concept drift and boolean noise-free examples
-    as described in Gama, João, et al [1]_.
+    This generator is an implementation of a data stream with abrupt concept drift and boolean
+    noise-free examples as described in Gama, João, et al [1]_.
 
-    It has four relevant attributes, two boolean attributes :math:`v, w` and two numeric attributes :math:`x, y`
-    uniformly distributed from 0 to 1. The examples are labaled depending on the classification function chosen
-    from below.
+    It has four relevant attributes, two boolean attributes :math:`v, w` and two numeric
+    attributes :math:`x, y` uniformly distributed from 0 to 1. The examples are labaled depending
```

Comment:
There is a typo in 'labaled' -> 'labeled'. Could you change it?

Reply:
Sure!